server/src/main/java/org/elasticsearch/ingest/IngestService.java

```diff
@@ -270,26 +271,49 @@ public static void resolvePipelinesAndUpdateIndexRequest(
         final IndexRequest indexRequest,
         final Metadata metadata
     ) {
-        resolvePipelinesAndUpdateIndexRequest(originalRequest, indexRequest, metadata, System.currentTimeMillis());
+        resolvePipelinesAndUpdateIndexRequest(originalRequest, indexRequest, metadata, Map.of());
+    }
+
+    public static void resolvePipelinesAndUpdateIndexRequest(
+        final DocWriteRequest<?> originalRequest,
+        final IndexRequest indexRequest,
+        final Metadata metadata,
+        Map<String, ComponentTemplate> templateSubstitutions
```

Comment:
Should we call this var `componentTemplateSubstitutions` since that's the name we've used elsewhere? (and presumably we'll have to add one for index template substitutions eventually)

Reply:
Yes definitely. I missed that one when I renamed all the rest.